aviatrix/resource_aviatrix_spoke_gateway_subnet_group.go

```diff
@@ -0,0 +1,161 @@
+package aviatrix
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/AviatrixSystems/terraform-provider-aviatrix/v2/goaviatrix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAviatrixSpokeGatewaySubnetGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateWithoutTimeout: resourceAviatrixSpokeGatewaySubnetGroupCreate,
+		ReadWithoutTimeout:   resourceAviatrixSpokeGatewaySubnetGroupRead,
+		UpdateWithoutTimeout: resourceAviatrixSpokeGatewaySubnetGroupUpdate,
+		DeleteWithoutTimeout: resourceAviatrixSpokeGatewaySubnetGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Subnet group name.",
+			},
+			"spoke_gateway_name": {
```

Comment:
Since resource name is aviatrix_spoke_gateway_subnet_group, here we can use "gw_name", right?